server/views/pages/optOut/optOut.njk

```diff
@@ -16,21 +15,25 @@
       },
       id: "otherReason",
       name: "otherReason",
+      value: submittedForm.otherReason,
       classes: "govuk-input--width-30",
       errorMessage: validationErrors | findError('otherReason')
     }) }}
 {% endset %}
 
 {% block content %}
     <div class="govuk-width-container">
+      <div class="govuk-grid-row">
         <div class="govuk-grid-column-two-thirds">
+            {% include 'partials/errorSummary.njk' %}
             {{ offenderSummary(assessment.prisonNumber, assessment.hdced, assessment.dateOfBirth, assessment.name) }}
             <div class="govuk-grid-row govuk-!-margin-top-8">
                 <form method="POST">
                     <input type="hidden" name="_csrf" value="{{ csrfToken }}">
                     {{ govukRadios({
-                        idPrefix: "optOutRadio",
+                        idPrefix: "optOutReason",
                         name: "optOutReason",
+                        value: submittedForm.optOutReason,
```

Comment:
realised doing the check questions that you can use `value` rather than individual `checked`s for radio button

Reply:
there's probably some requirement to show the previously saved state but it's a bit wishy washy at the moment so think we can hold back off that. (will require some api changes too)